modules/vaos/app/controllers/vaos/visits_controller.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module VAOS
+  class VisitsController < ApplicationController
+    SCHEDULE_TYPES = %w[direct request].freeze
+
+    def index
+      response = systems_service.get_facility_visits(system_id, facility_id, type_of_care_id, schedule_type)
+      render json: VAOS::FacilityVisitSerializer.new(response)
+    end
+
+    private
+
+    def systems_service
```

Comment:
I don't have any strong feelings on this, but i looks strange to me.

Why not just something like 
````ruby
def visit_params
  params.require([:system_id, :facility_id, :type_of_care_id, :schedule_type, :type_of_care_id])
end
````

Reply:
I agree, could do it that way -- what's problematic is when you then try to chain permit on the array, but just saying what is required is not a problem.

I think I'll fix this when we go in and refactor. There's a lot of cleanup ahead to make the work we've done more readable and consistent.